### DISCUSSION TITLE
43회차 반성문

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#430] [6. 반성문]
아이고 저번 주는 한참 롤토체스에 빠져서 블로그 쓰는 걸 깜빡했네요... 이번 주 부터 다시 열심히 써보겠습니다!
요즘 또 개발 욕구가 올라오고 있는데, 혹시 같이 개발하실 분 구합니다~

**Comment by 02ggang9** [comment]
저요저요저요저요저요저요

**Comment by shkisme** [comment]
저요저요저요저요
